.github/workflows/ci.yml

```diff
@@ -10,7 +10,19 @@ on:
     - cron:  '0 9 * * *'
 
 jobs:
+  run-rubocop:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
```

Comment:
@tannalynn What do you think about having this match the `TargetRubyVersion` defined in `.rubocop.yml`? It's currently set to 2.5. Perhaps keep this at 2.7 and bump up the `.rubocop.yml` one?

Reply:
Alright, I updated the .rubocop.yml to 2.7! thank you!

Reply:
I really like the ideas of these matching. Nice recommendation, James! Is this something we could set through a constant shared between the two files to reduce the toil of updating the version?

Reply:
RuboCop will send `.rubocop.yml` through an ERB parse before handing the content off to YAML, so there we could do something dynamic, such as:

```yaml
  TargetRubyVersion: <%= ENV.fetch('RUBY_VERSION', '2.7') %>
```

On the `ci.yml` side while trying to share an approach with RuboCop, we face these issues:

* The `ci.yml` parser won't have access to the source code locally
* The `ci.yml` content will be ran through a JavaScript parse instead of a Ruby one

I suppose we could take the approach I did with `test/helpers/ruby_rails_mappings.rb`, which uses `ci.yml` as a single source of truth. We could define the Ruby version once in `ci.yml` and then have RuboCop parse it out of there.

Reply:
I think I'd rather move forward with merging this in, but keep this improvement in mind for the future. Thanks for walking through some of the options on how this might be achieved.